src/oss/confluent.sh

```diff
@@ -1433,6 +1489,52 @@ version_command() {
     fi
 }
 
+produce_usage() {
+    cat <<EOF
+Usage: ${command_name} <topicname> [--value-format avro] [other optional args]
+
+Description:
+    Produce to Kafka topic specified by <topicname>.
+
+    Without '--value-format avro', it calls the 'kafka-console-producer' command,
+    automatically with '--topic <topicname> --broker-list localhost:9092' and passes in <other optional args>.
+
+    With '--value-format avro', it calls the 'kafka-avro-console-producer' command,
+    automatically with '--topic <topicname> --broker-list localhost:9092' and passes in <other optional args>.
+    This requires an additional argument '--property value.schema=<schema>'
+
```

Comment:
Should also tell the user what to do next, namely to type in the message contents (and key?) and hit enter

Reply:
Seems like a bit beyond MVP, given that current behavior has parity with underlying tools.

Reply:
Given my suggestion above to skip mention to optional args we can list as two options the `--property value.schema` and `--property key.schema` as a starting point. As @ybyzek this can be enhanced in a later version. But I'm in favor of adding `--property key.schema` as optional argument in the command as well as with an example here.

Reply:
I consider how to enter the message (and what is considered a message delimiter) as basic usage instruction.

Reply:
Added explanation to usage.